src/operations/command.ts

```diff
@@ -173,6 +179,18 @@ export abstract class CommandOperation<T> extends AbstractOperation<T> {
       }
     }
 
+    // if an API version was declared, add the apiVersion option to every command, except:
+    // a. only in the initial command of a transaction
+    // b. only in a Cursor's initiating command, not subsequent getMore commands
+    if (
+      server.serverApi &&
+      (!inTransaction || this.session.transaction.isStarting) &&
+      !cmd.commitTransaction &&
+      !cmd.getMore
+    ) {
+      applyServerApiVersion(cmd, server.serverApi);
```

Comment:
Note that this catches most operations that need the version applied to them, but not all, so there are some additional calls in this PR to the `applyServerApiVersion` helper for commands that don't use this class.